src/status_im/data_store/chats.cljs

```diff
@@ -1,42 +1,69 @@
 (ns status-im.data-store.chats
-  (:require [cljs.core.async :as async]
+  (:require [goog.object :as object]
+            [cljs.core.async :as async]
             [re-frame.core :as re-frame]
-            [status-im.data-store.realm.core :as core]
-            [status-im.data-store.realm.chats :as data-store])
-  (:refer-clojure :exclude [exists?]))
+            [status-im.data-store.messages :as messages]
+            [status-im.data-store.realm.core :as core]))
+
+(defn- normalize-chat [{:keys [chat-id] :as chat}]
+  (let [last-clock-value (-> (core/get-by-field @core/account-realm
+                                                :message :chat-id chat-id)
+                             (core/sorted :clock-value :desc)
+                             (core/single-clj :message)
+                             :clock-value)]
+    (assoc chat :last-clock-value  (or last-clock-value 0))))
```

Comment:
why two whitespaces between `:last-clock-value` and `(or` (i'm not mad, just accidentally noticed :D), but i suppose that's because you turned off auto re-indent

Reply:
Fixing